package.json

```diff
@@ -10,8 +10,8 @@
     "watch:css": "npx tailwindcss -i pkg/web/static/styles/style.css -o pkg/web/static/styles/output.css --watch"
   },
   "devDependencies": {
-    "daisyui": "^4.6.1",
-    "tailwindcss": "^3.4.1"
+    "daisyui": "^4.12.10",
```

Comment:
I noticed some bug fixes in the daisyui changelog, so it felt like a good time to upgrade.